### PR TITLE
Enable the IMU FTs in iCubGenova09

### PR DESCRIPTION
**Pull request by GiulioRomualdi**
This PR enables the streaming of the FTs IMU on iCubGenova09 everything seems working however we (with @isorrentino) notice this warning appearing. 
I think it is due to the imu mounted on one of the right foot FTs

```
[WARNING]  from BOARD 10.0.1.12 (right_leg-eb12-j3_5), src LOCAL, adr 0, time 80s 862m 356u: (code 0x00000039, par16 0x0005 par64 0x0000000000002000) -> SYS: a service has detected that some CAN boards are not broacasting anymore. In par16 the type of service, in par64 LS 32 bits the bit mask of lost board (CAN1 in MS 16 bits and CAN2 in LS 16 bits) + .
```

### CODE REVIEW

**Comment by traversaro** [comment]
Related code:
* https://github.com/robotology/icub-firmware/blob/f95bf9f525ea92d76974c887cde1855629918b3d/emBODY/eBcode/arch-arm/board/ems004/appl/v2/src/eoappservices/EOtheInertials3.c#L1810

**Review by S-Dafarra — APPROVED** [review]

File: iCubGenova09/hardware/inertials/right_arm-eb3-IMU.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE devices PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+    <device xmlns:xi="http://www.w3.org/2001/XInclude" name="right_arm-eb1-imu" type="embObjIMU">
```
Comment (GiulioRomualdi) [inline iCubGenova09/hardware/inertials/right_arm-eb3-IMU.xml]:
```suggestion
    <device xmlns:xi="http://www.w3.org/2001/XInclude" name="right_arm-eb3-imu" type="embObjIMU">
```

File: iCubGenova09/wrappers/inertials/right_arm-IMU_wrapper.xml [hunk 2]
```diff
@@ -2,20 +2,19 @@
...
+
 
-        
         <action phase="startup" level="5" type="attach">
             <paramlist name="networks">
                 <!-- The name of the element can be any string (we use SetOfInertials to better express its nature).
                      Its value must match the device name in the corresponding body_part-jx_y-inertials.xml file
                      or in body_part-ebX-inertials.xml -->
-                <elem name="SetOfInertials"> right_leg-eb11-inertials </elem>
+                <elem name="SetOfInertials">right_arm-eb1-imu </elem>
```
Comment (GiulioRomualdi) [inline iCubGenova09/wrappers/inertials/right_arm-IMU_wrapper.xml]:
```suggestion
                <elem name="SetOfInertials">right_arm-eb3-imu </elem>
```

**Review by isorrentino — APPROVED** [review]

**Comment by S-Dafarra** [comment]
@GiulioRomualdi let me know when you want to merge

**Comment by GiulioRomualdi** [comment]
@S-Dafarra fell free to merge it

**Pull request merged**